less/admin/ExtensionWidget.less

```
@@ -0,0 +1,58 @@
+.ExtensionsWidget-title {
+  color: @muted-color;
+}
+
+.ExtensionsWidget-list {
+  > .container {
+    padding: 0;
+  }
+}
+.ExtensionGroup {
+  margin-bottom: 20px;
+
+  h3 {
+    color: @muted-color;
+    text-transform: uppercase;
+    font-size: 12px;
+    margin: 0 0 10px;
+  }
+}
+
+.ExtensionList {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  .clearfix();
+
+  > li {
+    float: left;
+    text-align: left;
+    position: relative;
+  }
+}
+.ExtensionListItem.disabled {
+  .ExtensionListItem-title {
+    opacity: 0.5;
+    color: @muted-color;
+  }
+  .ExtensionListItem-icon {
+    opacity: 0.5;
+  }
+}
+.ExtensionListItem {
+  width: 90px;
+  margin-right: 15px;
+  margin-top: 10px;
+}
+
+.ExtensionIcon {
+  width: 90px;
+  height: 90px;
+  background: @control-bg;
+  color: @control-color;
+  border-radius: 6px;
+  display: inline-block;
+  font-size: 45px;
+  line-height: 90px;
+  text-align: center;
```

Let's also align the icon with flexbox! (it makes maintaining code and making design changes a lot easier in the long run)

```suggestion
  display: inline-flex;
  font-size: 45px;
  text-align: center;
  align-items: center;
  justify-content: center;
```